### DISCUSSION TITLE
debian build problems

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#2222] [Q&A] [answered]
I built re/rem/baresip packages from today's mains and when I start baresip, I get many errors on console.  The first one is
```
dl: mod: /usr/lib/x86_64-linux-gnu/baresip/modules/stdio.so (/usr/lib/x86_64-linux-gnu/baresip/modules/stdio.so: undefined symbol: ui_unregister)
```
Has something changed recently or is this my mistake?

**Comment by juha-h** [comment]
I reverted back to non-cmake debian/rules and the errors disappeared.

**Comment by sreimers** [comment]
I think I know whats happening. I tried to keep the old behavior for packaging and made baresip executable independent from libbaresip.so. I think I will revert this and repackage instead.

**Comment by sreimers** [comment, marked as answer]
Can you try: https://github.com/baresip/baresip/pull/2224

**Comment by juha-h** [comment]
Yes, https://github.com/baresip/baresip/pull/2224 fixed the problems.
